Tropos/Sources/main.m

```diff
@@ -1,8 +0,0 @@
-#import "TRAppDelegate.h"
```

Comment:
Did you mean to remove this?

Reply:
Yep, this is superseded by the `@UIApplicationMain` annotation in `AppDelegate.swift`.